.github/workflows/helm-lint-and-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: helm
+
+on:
+  pull_request:
+    paths:
+      - "charts/perses/**"
+    branches:
+      - main
+
+jobs:
+  helm-lint-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: setup
+        uses: ./.github/actions/setup_helmlint
+        with:
+          create-kind-cluster: "true"
+      - name: run chart-testing (install)
+        run: ct install --charts charts/perses
+      - name: install mdox
+        run: go install github.com/bwplotka/mdox@latest
```

Comment:
I don't get how `go` has been installed and I'm wondering if it is the explanation about why `mdox` is not found